project.clj

```diff
@@ -5,14 +5,14 @@
   :description "Liikenneviraston Harja"
 
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/clojurescript "1.8.51"]
+                 [org.clojure/clojurescript "1.9.293"]
 
                  ;;;;;;; Yleiset ;;;;;;;
-                 [prismatic/schema "1.0.4"]
+                 [prismatic/schema "1.1.3"]
```

Comment:
Vältetään: WARNING: Inst already refers to: cljs.core/Inst being replaced by: schema.core/Inst at line 394 dev-resources/js/out/schema/core.cljs